[Fixed] Warnings are output when building csharp template

<!--- Provide a general summary of the issue in the Title above -->

## Expected Behaviour
<!--- If you're describing a bug, tell us what should happen -->
<!--- If you're suggesting a change/improvement, tell us how it should work -->
No errors are present during build of C# function from template

## Current Behaviour
<!--- If describing a bug, tell us what happens instead of the expected behavior -->
<!--- If suggesting a change/improvement, explain the difference from current behavior -->
An error occurs during the `faas build` when using the `csharp` template

```
Program.cs(25,13): warning CS0436: The type 'FunctionHandler' in '/root/src/function/FunctionHandler.cs' conflicts with the imported type 'FunctionHandler' in 'Function, Version=0.0.0.0, Culture=neutral, PublicKeyToken=null'. Using the type defined in '/root/src/function/FunctionHandler.cs'. [/root/src/root.csproj]
```

## Possible Solution
<!--- Not obligatory, but suggest a fix/reason for the bug, -->
<!--- or ideas how to implement the addition or change -->
Remove the extra `FunctionHandler.csproj` file

## Steps to Reproduce (for bugs)
<!--- Provide a link to a live example, or an unambiguous set of steps to -->
<!--- reproduce this bug. Include code to reproduce, if relevant -->
1. `faas new tester --lang csharp`
2. `faas build -f tester.yml`
3. Observe the error text in the build output

## Context
<!--- How has this issue affected you? What are you trying to accomplish? -->
<!--- Providing context helps us come up with a solution that is most useful in the real world -->
Errors indicate things aren't working properly. If a new user saw this, or a sysadmin, they would think something is wrong with their code, but it is actually being caused by the template

## Your Environment
<!--- Include as many relevant details about the environment you experienced the bug in -->
* Docker version `docker version` (e.g. Docker 17.0.05 ):
17.12.0-ce

* Are you using Docker Swarm or Kubernetes (FaaS-netes)?
Both, but for this test in particular: Swarm

* Operating System and version (e.g. Linux, Windows, MacOS):
Ubuntu 17.10

* Link to your project or a code example to reproduce issue:
Just the csharp template code

--- a/template/csharp/root.csproj
+++ b/template/csharp/root.csproj
@@ -1,9 +1,14 @@
 <Project Sdk="Microsoft.NET.Sdk">
-
   <ItemGroup>
     <ProjectReference Include="function\Function.csproj" />
   </ItemGroup>
+  <PropertyGroup>
+      <EnableDefaultCompileItems>false</EnableDefaultCompileItems>
+  </PropertyGroup>
 
+  <ItemGroup>
+    <Compile Include="./Program.cs"></Compile>
+  </ItemGroup>
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp2.0</TargetFramework>

